js/script.js

```diff
@@ -1,10 +1,174 @@
+
+        taskController.addTask(task);
+        storage.updateStorageTasks(taskController.getTasks());
+        taskController.renderTasks(storage.getStorageSorting());
+
+        resetForm(form_add, "#addModal");
+    } else {
+        alert("Please, input all fields!");
+    }
+
+});
+
+
+//------------------ Edit Task Button Logic --------------------
+btn_editTask.addEventListener("click", (e) => {
+    e.preventDefault();
+
+    const id = document.querySelector("#task_id_edit").value;
+    const title = input_title_edit.value,
+        text = input_text_edit.value,
+        priority = document.querySelector("input[name=priorityRadios_edit]:checked").value;
+
+    if (title && text && priority) {
+        taskController.updateTask({ id, title, text, priority });
+        storage.updateStorageTasks(taskController.getTasks());
+        taskController.renderTasks(storage.getStorageSorting());
+
+        resetForm(form_edit, "#editModal");
+    } else {
+        alert("Please, input all fields!");
+    }
+});
+
+
+function resetForm(form, modal) {
+    $(modal).modal('hide');
+    form.reset();
+}
+
+
+
+//------------------ Complete/Edit/Delete Task Buttons Logic --------------------
+document.querySelector(".main-content").addEventListener("click", (event) => {
+    const target = event.target;
+
+    if (target.classList.contains("btn_completeTask")) {
+        const taskId = target.dataset.id;
+        taskController.completeTask(taskId);
+        update();
+    }
+    else if (target.classList.contains("btn_editTask")) {
+        const taskId = target.dataset.id;
+        taskController.updateEditModal(taskId);
+        update();
+    }
+    else if (target.classList.contains("btn_deleteTask")) {
+        const taskId = target.dataset.id;
+        taskController.deleteTask(taskId);
+        update();
+    }
+
+    function update() {
+        storage.updateStorageTasks(taskController.getTasks());
+        taskController.renderTasks(storage.getStorageSorting());
+    }
+});
+
+
+//------------------ Sorting Tasks Buttons Logic --------------------
+if (storage.getStorageSorting() === 'asc') {
+    tooglePrimaryBtn(btn_asc, btn_desc);
+} else {
+    tooglePrimaryBtn(btn_desc, btn_asc);
+}
+
+btn_asc.addEventListener("click", () => {
+    storage.setStorageSorting(0);
+    taskController.renderTasks(storage.getStorageSorting());
+    tooglePrimaryBtn(btn_asc, btn_desc);
+});
+
+btn_desc.addEventListener("click", () => {
+    storage.setStorageSorting(1);
+    taskController.renderTasks(storage.getStorageSorting());
+    tooglePrimaryBtn(btn_desc, btn_asc);
+});
+
+function tooglePrimaryBtn(btn_primary, btn_secondary) {
+    btn_primary.classList.remove("btn-secondary");
+    btn_primary.classList.add("btn-primary");
+
+    btn_secondary.classList.remove("btn-primary");
+    btn_secondary.classList.add("btn-secondary");
+}
+
+
+//------------------ Theme Switching Logic --------------------
```

Comment:
could you create a module for this ?

Reply:
created a module for this
```suggestion
import ThemeSwitcher from "./theme-switcher.js";
```